blocks/api/utils.js

```diff
@@ -25,10 +30,11 @@ export function isUnmodifiedDefaultBlock( block ) {
 	}
 
 	const newDefaultBlock = createBlock( defaultBlockName );
-	const attributeKeys = [
+
+	const attributeKeys = applyFilters( 'blocks.unmodifiedBlockAttributes', [
```

Comment:
Need some naming convention for these. I've been doing roughly `[module].[function].[modifier]`. Thoughts?

Reply:
I like this convention, updating